lib/contentful.js

```diff
@@ -7,11 +7,10 @@
 
 import assign from 'lodash/assign'
 import axios from 'axios'
-import version from '../version'
 import {createHttpClient} from 'contentful-sdk-core'
 import createContentfulApi from './create-contentful-api'
 import createLinkResolver from './create-link-resolver'
-
+import {getUserAgentHeader} from './utils/user-agent'
```

Comment:
It might be smart do add this function as a generic one to `contentful-sdk-core`?